src/components/calcite-chip/calcite-chip.tsx

```diff
@@ -79,28 +79,70 @@ export class CalciteChip {
         : "l";
 
     const iconEl = (
-      <calcite-icon
-        class="calcite-chip--icon"
-        icon={this.icon}
-        scale={iconScale}
-      />
+      <calcite-icon class={this.iconClass} icon={this.icon} scale={iconScale} />
```

Comment:
The classes for the `icon`, `span`, and `button` elements were all pretty long, and would've spanned multiple lines, so I split them into their own functions.